include/votca/csg/topology.h

```diff
@@ -426,7 +459,7 @@ class Topology {
 
   std::map<std::string, Index> _interaction_groups;
 
-  std::map<std::string, std::list<Interaction *> > _interactions_by_group;
+  std::map<std::string, std::list<const Interaction *>> _interactions_by_group;
```

Comment:
same here. I think up to 50000 `int` or pointers std::vector is much faster than list

Reply:
Ok I'll try to make the switch.